docs/docs/getting_started.md.in

```diff
@@ -6,15 +6,14 @@ builds each requested plugin, builds the main ILLIXR binary, and (optionally) in
 ILLIXR currently only builds on Linux systems, and has been tested on the following configurations:
 
 - Ubuntu
-    - 20.04
+    - 20.04[^1]
     - 22.04
+    - 24.04
 - Fedora[^1]
```

Comment:
This footnote has to be `[^2]`.

Reply:
I have fixed the footnote issue. As far as building the documentation, you need to enable it when running cmake (use -DBUILD_DOCS=ON)